news/2017-11-05-Official-Mapping-Contests:-osu!-Mapping-Olympiad.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post 
+title: "Official Mapping Contests: osu! Mapping Olympiad"
+date: 2017-11-05
+--- 
+
+![](filepath) BANNER IN PROGRESS
+
+After a long hiatus, Chaos, Voli and I are happy to announce the revival of official beatmapping contests: Welcome to the _osu! Mapping Olympiad!_
+
+Let me elaborate on the concept for the osu! Mapping Olympiad. To ensure that there is a healthy and consistent flow of official contests, 
+we have worked on a concept which involved all types of modes - three contests per mode will happen annually in a specific order. 
```

Comment:
remove new lines like these? they dont do much

Reply:
Why? I think it gives it a better structure (Introductory sentence / Short elaboration)